src/Components/Dropdowns/Composed/MenuDropdown.tsx

```diff
@@ -286,17 +296,16 @@ export const MenuDropdown: FC<MenuDropdownProps> = ({
   )
 
   const typeAheadMenu = () => {
-    const itemWidth = {width: 'calc(100% - 8px)'}
+    const itemStyle = {width: '100%', paddingLeft: '14px'}
```

Comment:
should these two inline styles be moved to the stylesheet? That way, it'll be less of a hassle to override them if they need to be

Reply:
Done!